.github/workflows/build-packages.yml

```diff
@@ -30,33 +30,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        architecture: [amd64, arm64]
+        architecture: [linux/amd64, linux/arm64]
```

Comment:
architecture may not the correct name `platform` will be better

Reply:
At this point I don't want to change it and risk breaking eveyrthing because I missed one small letter somewhere lol